lib/rubocop/cop/factory_bot/id_sequence.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module FactoryBot
+      # Do not create a FactoryBot sequence for an id column.
+      #
```

Comment:
I see that this cop is `Safe: false`. So, could you add that explanation as follows?
https://github.com/rubocop/rubocop-factory_bot/blob/afc310efc27661d673349aa560b3559dae1ab7b5/lib/rubocop/cop/factory_bot/ association_style.rb#L8-L11

Reply:
Hmm, on reflection, this cop probably _is_ safe, since it's only deleting some code, rather than adding anything (that may only work if a given module has been included). What do you think? If you agree, I'll change to `Safe: true`

Reply:
Actually, I just went with it being safe - let me know if you have any concerns and I can change it back.

Reply:
I too think `Safe: true` looks good. It was false and I asked why, but I too think this cop can be marked as safe 👍